assets/src/domain/eventEditor/ui/ticketAssignmentsManager/styles.scss

```
@@ -0,0 +1,18 @@
+#ticket-assignment-manager {
+	td,
+	th {
+		text-align: center;
+	}
+	.no-assignments {
+		background-color: var(--ee-color-hot-pink) !important;
```

awesome! Thanks for using vars!

you could also use:

```suggestion
		background-color: var(--ee-color-accent) !important;
```